hols/todo/jaeger.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker run --rm --name jaeger \
```

Comment:
This can be inlined in a README instead of in a script.

Reply:
It's convenient

Reply:
It's better to tell users to do `docker xyz` than `./foobar.sh`.

Use shell scripts for complicated steps, not simple straightforward commands that users would easily understand.

Reply:
removed